uk.co.compucorp.civicrm.hrleaveandabsences/views/shared/components/manage-leave-requests.html

```diff
@@ -93,7 +93,7 @@
         </div>
         <div class="checkbox">
           <label>
-            <input type="checkbox" ng-change = "vm.refresh()"
+            <input type="checkbox" ng-change = "vm.pendingRequestListener()"
```

Comment:
why can't you simply pass some "reset status filter to all" param to `refresh` instead of creating a whole new method?